index.html

```diff
@@ -1512,12 +1517,12 @@ <h3 id="applying">
           <a>navigate</a> to the <a>start URL</a> with <a>replacement
           enabled</a>.
         </p>
-        <div>
+        <div class="note">
           <p>
-            Please note that the <a>start URL</a> is not necessarily the value
-            of the <a data-link-for="WebAppManifest"><code>start_url</code></a>
-            member: the user or user agent could have changed it when the
-            application was added to home-screen or otherwise bookmarked.
+            The <a>start URL</a> is not necessarily the value of the
+            <a data-link-for="WebAppManifest"><code>start_url</code></a> member:
+            the user or user agent could have changed it when the application
+            was added to home-screen or otherwise bookmarked.
```

Comment:
"was added to...." -> "installed"?

Reply:
OK, moved this to #701 since it's kind of off-topic for this breaking change.